akka-persistence-query/src/main/scala/akka/persistence/query/scaladsl/CurrentEventsByTagQuery.scala

```diff
@@ -17,7 +17,7 @@ trait CurrentEventsByTagQuery extends ReadJournal {
    * is completed immediately when it reaches the end of the "result set". Events that are
    * stored after the query is completed are not included in the event stream.
    */
-  def currentEventsByTag(tag: String, offset: Long): Source[EventEnvelope, NotUsed]
+  def currentEventsByTag(tag: String, offset: Offset): Source[EventEnvelope, NotUsed]
```

Comment:
One topic for discussion is how we shall introduce this change without breaking plugins.
We can change the API, since persistence-qurey is experimental, but I think it can cause unnecessary trouble to do that in a 2.4.x. When we release Akka 2.5 we can change the API before removing the experimental label.

One way would be to add `CurrentEventsByTagQuery2` with the new signatures. Then in 2.5 we make that the only trait (and rename it to `CurrentEventsByTagQuery`).